container-guide/adoc/containers-verify.adoc

```diff
@@ -30,5 +26,165 @@ To automatically validate an image when you pull it, set the environment `DOCKER
 env DOCKER_CONTENT_TRUST=1 docker pull registry.suse.com/suse/sle15:latest
 ....
 
+[[sec-verify-cosign]]
+=== Verifying {bcia} with Cosign
+
+To verify a {bcia} image, run Cosign in the container. The command below fetches the
+signing key from the SUSE server and uses it to verify the latest BCI-Base container
+image.
+
+[source,ShellSession]
+----
+> podman run --rm -it gcr.io/projectsigstore/cosign verify \
+    --key https://ftp.suse.com/pub/projects/security/keys/container–key.pem \
+    registry.suse.com/bci/bci-base:latest | tail -1 | jq
+
+[
+  {
+    "critical": {
+      "identity": {
+        "docker-reference": "registry.suse.com/bci/bci-base"
+      },
+      "image": {
+        "docker-manifest-digest": "sha256:52a828600279746ef669cf02a599660cd53faf4b2430a6b211d593c3add047f5"
+      },
+      "type": "cosign container image signature"
+    },
+    "optional": {
+      "creator": "OBS"
+    }
+  }
+]
+----
+
+The signing key can be used to verify all {bcia} container images. The signing
+key ships with {sle} 15 (the `/usr/share/container-keys/suse-container-key.pem` file).
+
+You can also check {bcia} container images against
+https://github.com/sigstore/rekor[rekor], the immutable tamper resistant
+ledger. For example:
+
+[source,ShellSession]
+----
+> podman run --rm -it -e COSIGN_EXPERIMENTAL=1 gcr.io/projectsigstore/cosign \
+    verify --key https://ftp.suse.com/pub/projects/security/keys/container–key.pem \
+    registry.suse.com/bci/bci-base:latest | tail -1 | jq
+[
+  {
+    "critical": {
+      "identity": {
+        "docker-reference": "registry.suse.com/bci/bci-base"
+      },
+      "image": {
+        "docker-manifest-digest": "sha256:52a828600279746ef669cf02a599660cd53faf4b2430a6b211d593c3add047f5"
+      },
+      "type": "cosign container image signature"
+    },
+    "optional": {
+      "creator": "OBS"
+    }
+  }
+]
+----
+
+If verification fails, the output of the `cosign verify` command is similar to the one below.
+
+----
+Error: no matching signatures:
+crypto/rsa: verification error
+main.go:62: error during command execution: no matching signatures:
+crypto/rsa: verification error
+----
+
 [[sec-verify-podman]]
-==== Podman
+== Verifying {bcia} with {podman}
+
+Verifying {bcia}s using {podman} is a three-step process. First, specify
```

Comment:
Since you've mentioned "three-step", do you think it would help to indicate this somehow? At least from the source here, I can ascertain the second step but I can't figure out which is the 3rd(similar levels).

Reply:
Good point! I'll fix it.